### ISSUE TITLE
Donate Page_Image Button missing Alt text

### ISSUE THREAD

**Issue by HerinHentry** [mr-bobz/talarts#23]
![image](https://cloud.githubusercontent.com/assets/15111528/10796801/e44498a4-7df3-11e5-99b8-f364e22ed6db.png)

**Comment by HerinHentry** [comment]
Issue fixed.
Can be closed.
